src/mem/MemCtrl.py

```diff
@@ -89,4 +91,6 @@ class MemCtrl(QoSMemCtrl):
     static_frontend_latency = Param.Latency("10ns", "Static frontend latency")
     static_backend_latency = Param.Latency("10ns", "Static backend latency")
 
+    # latency for dram cache tag check
+    static_tagcheck_latency = Param.Latency("2ns", "Static tagcheck latency")
```

Comment:
The help string should be more than just repeating the name. (for instance, you could have put the comment on the line before as the help string)